### PR TITLE
Create Journal of Reconstructive Microsurgery

### PR DESCRIPTION
**Pull request by go2roland**
References must be listed double-spaced, numbered, in AMA style, using Index Medicus
journal title abbreviations, and in the same sequence as they are cited in the text.

By way of exception to AMA style, do not italicize book titles or journal title abbreviations and do not put a period at the end of a reference.

### CODE REVIEW

**Comment by rmzelle** [comment]
Better :).

Can you go to https://github.com/go2roland/styles/blob/patch-2/Journal%20of%20Reconstructive%20Microsurgery, use the pencil icon, and change the file name to "journal-of-reconstructive-microsurgery.csl"? See https://help.github.com/articles/renaming-a-file/ for additional instructions.

**Comment by go2roland** [comment]
I try to do my best :)

**Comment by rmzelle** [comment]
Great. The only complaint of our automated testing system is that your style contains two unused macros, "author-short" and "year-date". You can delete those by using the pencil icon at https://github.com/go2roland/styles/blob/patch-2/journal-of-reconstructive-microsurgery.csl

File: journal-of-reconstructive-microsurgery.csl [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      <name>Roland Bockmann</name>
+      <email>go2roland@gmx.net</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <summary>A Variation of the American Medical Association style as used in J Reconstr Microsurg.</summary>
+    <updated>2015-06-04T20:56:26+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
```
Comment (adam3smith) [inline journal-of-reconstructive-microsurgery.csl]: delete these lines 21 - 25. They set the page-range delimiter to a hyphen, the journal uses en dash (as it should!).

File: journal-of-reconstructive-microsurgery.csl [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+                  <date variable="accessed">
+                    <date-part name="month" suffix=" "/>
+                    <date-part name="day" suffix=", "/>
+                    <date-part name="year"/>
+                  </date>
+                </group>
+              </group>
+            </if>
+            <else>
+              <group delimiter=":" suffix=".">
```
Comment (adam3smith) [inline journal-of-reconstructive-microsurgery.csl]: remove the ```suffix="."``` here -- bib entries are not supposed to end in a period.

File: journal-of-reconstructive-microsurgery.csl [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    <id>http://www.zotero.org/styles/journal-of-reconstructive-microsurgery</id>
+    <link href="http://www.zotero.org/styles/journal-of-reconstructive-microsurgery" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
+    <link href="https://www.thieme.de/de/journal-reconstructive-microsurgery/authors-9770.htm" rel="documentation"/>
+    <author>
+      <name>Roland Bockmann</name>
+      <email>go2roland@gmx.net</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
```
Comment (adam3smith) [inline journal-of-reconstructive-microsurgery.csl]: please add the ISSNs here:

```
<issn>0743-684X</issn>
<eissn>1098-8947</eissn>
```

**Comment by adam3smith** [comment]
OK, I have some more comments that'll improve the style above, with those we're good to go. Once again,
 You can edit the style by using the pencil icon at https://github.com/go2roland/styles/blob/patch-2/journal-of-reconstructive-microsurgery.csl
Please post here briefly when you're done--we're not getting notifications for updated styles.

**Comment by go2roland** [comment]
Dear Sebastian,


Thanks for your kind comments. As you will hopefully see I did my best to get it done right.


Kind regards,
Roland

**Comment by adam3smith** [comment]
great, thanks!

**Comment by go2roland** [comment]
Hello,


Today I tried to use my own style Journal of Reconstructive Microsurgery (#1576) myself and I have to admit that it doen’t work at all. It only produces error messages in paper2.
Therefore I gave it another try and rebuilt it from “scratch”. 
Can you give me some hints, how can I change the script in github?


Best regards,


Roland

**Pull request merged**